test/EFCore.PG.FunctionalTests/Query/GearsOfWarQueryNpgsqlFixture.cs

```diff
@@ -41,8 +41,12 @@ public override ISetSource GetExpectedData()
             // Also chop sub-microsecond precision which PostgreSQL does not support.
             foreach (var mission in _expectedData.Missions)
             {
-                mission.Timeline = new DateTimeOffset(
-                    mission.Timeline.Ticks - (mission.Timeline.Ticks % (TimeSpan.TicksPerMillisecond / 1000)), TimeSpan.Zero);
+                long newTicks = mission.Timeline.Ticks - (mission.Timeline.Ticks % (TimeSpan.TicksPerMillisecond / 1000));
+                // Ensure the newTicks are within the DateTime valid range
+                if (newTicks < DateTime.MinValue.Ticks) newTicks = DateTime.MinValue.Ticks;
+                if (newTicks > DateTime.MaxValue.Ticks) newTicks = DateTime.MaxValue.Ticks;
+
+                mission.Timeline = new DateTimeOffset(newTicks, TimeSpan.Zero);
```

Comment:
I guess the CRDB side isn't happy when this happens?

Reply:
Yep!